.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.idea/
```

Comment:
This is okay to put here, but you might want to also just put this in your global gitignore so you don't need to do it on every project.

e.g. in your `~/.gitconfig` you would have this:

```
[core]
          excludesfile = /Users/erika/.gitignore-global
```

And then in `~/.gitignore-global` you would have something like this:

```
*.iml
*.ipr
*.iws
.idea
.DS_Store
```

Reply:
TIL 🙇